src/udf/node_class.c

```
@@ -0,0 +1,739 @@
+	const Node *node    // pointer to the native FalkorDB Node
+) {
+	ASSERT (node   != NULL) ;
+	ASSERT (js_ctx != NULL) ;
+
+	// clone node
+	Node *_node = rm_malloc (sizeof (Node)) ;
+	memcpy (_node, node, sizeof (Node)) ;
+
+    JSValue obj = JS_NewObjectClass (js_ctx, js_node_class_id) ;
+    if (JS_IsException (obj)) {
+        return obj ;
```

Cppcheck (2.18.0)

[error] Memory leak

(memleak)